src/userNotificationManager/userNotificationManager.ts

```diff
@@ -30,12 +31,15 @@ export default class UserNotificationManagerTransport extends winston.Transport
     metadata?: any,
     callback?: (arg1, arg2) => void
   ) {
+    const focusTaskOutput = idfConf.readParameter(
+      "idf.notificationSilentMode"
+    ) as boolean;
     if (metadata && metadata.user) {
-      if (level === "info") {
+      if (level === "info" && !focusTaskOutput) {
         vscode.window.showInformationMessage(message);
-      } else if (level === "warn") {
+      } else if (level === "warn" && !focusTaskOutput) {
         vscode.window.showWarningMessage(message);
-      } else if (level === "error") {
+      } else if (level === "error" && !focusTaskOutput) {
```

Comment:
Not on error, let keep error intact, and user should receive error notification always